src/LessMsi.Gui/MainFormPresenter.cs

```diff
@@ -378,7 +379,7 @@ public void OnSelectedStreamChanged()
 					var foundStream = oleFile.GetStreams().FirstOrDefault(s => string.Equals(View.SelectedStreamInfo.Name, s.Name, StringComparison.InvariantCulture));
 					if (foundStream == null)
 					{
-						View.ShowUserError("Could not find stream for CAB '{0}'", View.SelectedStreamInfo.Name);
+						View.ShowUserError($"{Strings.FindStreamError} CAB '{0}'", View.SelectedStreamInfo.Name);

```

Comment:
put the placeholder (`{0}`) in to the translation string - this was done appropriately in other places.